templates/chartmuseum/chartmuseum-sm.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.chartmuseum.enabled .Values.chartmuseum.serviceMonitor.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: "{{ template "harbor.chartmuseum" . }}"
+  labels:
+{{ include "harbor.labels" . | indent 4 }}
+    component: chartmuseum
+spec:
+  endpoints:
+    - interval: 10s
+      scrapeTimeout: 10s
```

Comment:
same here

Reply:
I maintained for the past 6 month a separated helm chart for the exporter. as it was just going to stall forever here